### ISSUE TITLE
Integration does not show up in HA Core Add Integration menu

### ISSUE THREAD

**Issue by davethetallguy** [vinteo/hass-opensprinkler#154]
Similar to [151](https://github.com/vinteo/hass-opensprinkler/issues/151) and [137](https://github.com/vinteo/hass-opensprinkler/issues/137) after loading the integration through HACS, it still does not appear on the Add Integration menu. I have tried restarting the server from Configuration -> Server Controls -> Server Management.  I have also restarted the docker container.  

The log suggests it is found by Home Assistant, even if not displayed, as this warning is presented:

- 2021-07-04 15:59:40 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration opensprinkler which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant

(This is the same warning all HACS modules, including HACS, generate at startup.)

Unlike the experience in the previous bugs, the integration has not appeared for configuration after several hours, after reboots, or after reboot and waiting several hours.  

Suggestions will be appreciated.

**Comment by vinteo** [comment]
Does clearing the browser cache help?

**Comment by davethetallguy** [comment]
So in keeping with my predecessors, I found a solution on my own.  

Although I rebooted within the application several times, the integration did not appear until I logged out of the application, then back in again. 

@vinteo I had not caught your comment prior to the logout. I didn't expressly clear the browser cache, but that would have been a good next step. I signed back in  the same tab immediately after signing out, but it appears to be an artifact of something cached in Safari.
